bindings/nodejs/lib/types/block/signature.ts

```diff
@@ -33,17 +33,24 @@ class Ed25519Signature extends Signature {
     /**
      * The public key.
      */
-    publicKey: HexEncodedString;
+    publicKey: Ed25519PublicKey;
```

Comment:
change to readonly and remove the getter? weve been switching to that style

Reply:
That's out of scope, we have an own issue for the signature

Reply:
Actually this change is wrong I think